### PR TITLE
Publish blog posts for Harold and network request panel

### PR DESCRIPTION
**Pull request by ccschmitz**
## Summary

Adds two blog posts:

1. [Announcing Harold](https://docs.google.com/document/d/16b4mf1HkfRfvH8-5Fn-DNBs_6hH20wyDxl_H-bPqj0E/edit)
2. [Network Resource Panel](https://docs.google.com/document/d/166Wa8CqbLNu7-MsFVA7mUa3YlZ7zSY-DqRxhLw3aE-4/edit)

## How did you test this change?

Vercel preview.

## Are there any deployment considerations?

N/A

### CODE REVIEW

**Comment by ccschmitz** [comment]
@jay-khatri @deltaepsilon would appreciate your feedback on this whenever you have a minute! I didn't have URLs for some of the upcoming posts so I made up URLs for them. Would appreciate help coordinating to make sure I'm not publishing broken links 😅 - feel free to revise as you see fit!

Also, let me know if there's any imagery I missed or that you want to swap in. I just pushed up a change that uses all the assets from Julian rather than the screenshots I took from the app.

File: blog-content/introducing-harold.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+After generating a suggestion on how to fix an error, you’ll be prompted to provide some feedback to help us understand whether these suggestions are useful. This feedback is critical to help us improve Harold’s capabilities, so we appreciate the time you take to share as it will help shape our roadmap for our AI capabilities going forward.
+
+## Ask Harold for Session Insights
+
+Ever wonder what to do with that 4 hour long session? Harold can now analyze the events in a session and summarize them for you, providing a handful of insights about what the user did in that session. This can help with quickly understanding their flow in the app and allow you to quickly jump to the interesting portions of the session.
+
+![Harold suggesting how to fix an error](/images/blog/introducing-harold/session-insights.png)
+
+These insights, paired with [Interesting Sessions](https://highlight.io/interesting-sessions), power a new weekly email digest that summarizes the interesting things happening in your product so you can quickly see new or unexpected usage patterns.
```
Comment (jay-khatri) [inline blog-content/introducing-harold.md:35]:
```suggestion
These insights, paired with [Interesting Sessions](https://highlight.io/blog/interesting-sessions), power a new weekly email digest that summarizes the interesting things happening in your product so you can quickly see new or unexpected usage patterns.
```

just replacing with a reasonable slug for zane's blog post.

File: blog-content/introducing-harold.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## What’s Next for Harold
+
+We hope to give Harold more superpowers soon, especially if our users are finding a lot of value from these initial features. Please let us know what you think of these features and if you find them useful!
+
+Beyond these AI features, we have a [slew of other updates](https://highlight.io/launch-week) to help you wrangle errors and get to the most important user events in your sessions. Check out our new [auto-resolving errors functionality](https://highlight.io/auto-resolving-errors), option for [filtering errors from browser extensions](https://highlight.io/filtering-browser-extensions-errors), and [interesting sessions](https://highlight.io/interesting-sessions)!
```
Comment (jay-khatri) [inline blog-content/introducing-harold.md:41]:
```suggestion
Beyond these AI features, we have a [slew of other updates](https://highlight.io/launch-week) to help you wrangle errors and get to the most important user events in your sessions. 
```

I would remove the second sentence here for simplicity.

**Review by jay-khatri — APPROVED** [review]

**Comment by jay-khatri** [comment]
One more thing. We should add the launch week 2 label:
https://highlightcorp.slack.com/archives/C059S8A48MD/p1689360004882219

**Pull request merged**